lib/takeaway.rb

```diff
@@ -0,0 +1,31 @@
+require_relative 'menu'
+require_relative 'order'
+
+class Takeaway  
+  def initialize
+    @menu = Menu.new
```

Comment:
Why do both Takeaway and Order need an instance of Menu?  I assume that it's so Order can check that a given item is on the Menu but... it's another point of coupling in the program.